nanoFramework.Json/JsonConvert.cs

```diff
@@ -90,8 +90,8 @@ public static object DeserializeObject(StreamReader dr, Type type)
         private static object Deserialize(StreamReader dr)
         {
             // Read the DataReader into jsonBytes[]
-            jsonBytes = new byte[dr.BaseStream.Length];
-            jsonPos = 0;
+            var jsonBytes = new byte[dr.BaseStream.Length];
```

Comment:
hum, where were they defined before the change?

Reply:
In class as static field. Check line 768-769 in old file.

Reply:
so then they should be removed if you instantiate them locally every time?

Reply:
They are, that's why I wrote "768-769 in **old file**" ;)